### DISCUSSION TITLE
Additional formatting for CONCAT formula output [Jun 25]

### DISCUSSION THREAD

**Discussion by dstala** [nocodb/nocodb#6005] [Discord Threads]
> 🙋‍♂️ **nachteulen.coffee**

**Comment by o1lab** [comment]
> 👨‍🦲 **dstala**  
#### Not supported currently.  
#  

 [Autogenerated from discord](https://discord.com/channels/661905455894888490/1111292184382165102)  
- - - -  
#### Join [NocoDB's](http://www.nocodb.com/)  community  
[Discord](https://discord.gg/5RgZmkW) • [Twitter](https://twitter.com/nocodb) • [Reddit](https://www.reddit.com/r/NocoDB/) • [Docs](https://docs.nocodb.com/)
